integration/create_test.go

```diff
@@ -241,3 +243,102 @@ func TestCreateStressSameCollection(t *testing.T) {
 
 	require.Equal(t, int32(1), created.Load(), "Only one attempt to create a collection should succeed")
 }
+
+// TestCreateCappedCommandInvalidSpec checks that invalid create capped collection commands are handled correctly.
+// For valid test cases see collStats for capped collections tests.
+func TestCreateCappedCommandInvalidSpec(t *testing.T) {
+	t.Parallel()
+
+	for name, tc := range map[string]struct {
+		collectionName any
+		capped         any
+		size           any
+		max            any
+
+		err        *mongo.CommandError // required, expected error from MongoDB
+		altMessage string              // optional, alternative error message for FerretDB, ignored if empty
+	}{
+		"NoSize": {
+			collectionName: "no_size",
+			capped:         true,
+			err: &mongo.CommandError{
+				Code:    72,
+				Name:    "InvalidOptions",
+				Message: "the 'size' field is required when 'capped' is true",
+			},
+		},
+		"NoSizeWithMax": {
+			collectionName: "no_size_with_max",
+			capped:         true,
+			max:            500,
+			err: &mongo.CommandError{
+				Code:    72,
+				Name:    "InvalidOptions",
+				Message: "the 'size' field is required when 'capped' is true",
+			},
+		},
+		"WrongSizeType": {
+			collectionName: "wrong_size",
+			capped:         true,
+			size:           "foo",
+			err: &mongo.CommandError{
+				Code:    14,
+				Name:    "TypeMismatch",
+				Message: "BSON field 'create.size' is the wrong type 'string', expected types '[long, int, decimal, double']",
+			},
+		},
+		"WrongMaxType": {
+			collectionName: "wrong_max",
+			capped:         true,
+			size:           500,
+			max:            "foo",
+			err: &mongo.CommandError{
+				Code:    14,
+				Name:    "TypeMismatch",
+				Message: "BSON field 'create.max' is the wrong type 'string', expected types '[long, int, decimal, double']",
+			},
+		},
+		"WrongCappedType": {
+			collectionName: "wrong_capped",
+			capped:         "foo",
+			err: &mongo.CommandError{
+				Code:    14,
+				Name:    "TypeMismatch",
+				Message: "BSON field 'create.capped' is the wrong type 'string', expected types '[bool, long, int, decimal, double']",
+			},
+		},
+		"NegativeSize": {
```

Comment:
Let's add a test for zero size, I recall it returns error too

Reply:
Great idea! Added a test case.